Detectors/GlobalTrackingWorkflow/src/tof-matcher-workflow.cxx

```diff
@@ -123,6 +126,23 @@ WorkflowSpec defineDataProcessing(ConfigContext const& configcontext)
 
   specs.emplace_back(o2::globaltracking::getTOFMatcherSpec(src, useMC, useFIT, false, strict)); // doTPCrefit not yet supported (need to load TPC clusters?)
 
+  // initialize collision context
+  if (gSystem->AccessPathName("collisioncontext.root")) {
+    LOG(info) << "collisioncontext.root not available, let's skip it (cosmics?) ";
+  } else {
+    auto mcReader = std::make_unique<o2::steer::MCKinematicsReader>("collisioncontext.root");
+    auto context = mcReader->getDigitizationContext();
+    if (context) {
+      auto bcf = context->getBunchFilling();
+      std::bitset<3564> isInBC = bcf.getBCPattern();
+      for (unsigned int i = 0; i < isInBC.size(); i++) {
+        if (isInBC.test(i)) {
+          o2::tof::Utils::addInteractionBC(i, true);
+        }
+      }
+    }
+  }
+
```

Comment:
Note that in general the bunch filling will not be available in the workflow definition but should be extracted from the GRP at the 1st TF processing. At the moment merging as is.